web/views/pages/index.ejs

```diff
@@ -5,10 +5,14 @@
         <meta name="description" content="Electricity Map is a live visualization of where your electricity comes from and how much CO2 was emitted to produce it." />
         <meta property="og:description" content="Electricity Map is a live visualization of where your electricity comes from and how much CO2 was emitted to produce it." />
         <meta property="og:image" content="https://cloud.githubusercontent.com/assets/1655848/20340757/5ada5cf6-abe3-11e6-97c4-e68929b8a135.png" />
-        <meta property="og:title" content="Electricity Map | Live CO2 emissions of the European electricity consumption" />
+        <meta property="og:title" content="<%= __('misc.maintitle') %>" />
         <meta property="og:type" content="website" />
```

Comment:
I don t know if that on purpose but you don t have anymore Electricity map | before the title